backend/src/models/items-by-tag.model.ts

```diff
@@ -0,0 +1,25 @@
+import { AggregateRoot } from '@nestjs/cqrs';
+import { prop } from '@typegoose/typegoose';
+import { Expose } from 'class-transformer';
+import { Item } from './item.model';
+
+// TODO: make sure the indexes are not taken from the Item model
+export class ItemsByTag extends AggregateRoot {
+  @Expose()
+  @prop({ required: true, unique: true })
+  tagName: string;
+
+  @Expose()
+  @prop({
+    array: true,
+    type: () => [Item],
+    excludeIndexes: true,
+    _id: false,
```

Comment:
Why with out id?

Reply:
coz read db and read db does not have references and our name already is an index and we use it as an id so we don't need more "useless" Code

Reply:
But Indeed it is an issue that we Excude all indexes, because we would like to have the index in the tag of the item persist but not the slug in the item :/